apps/src/lib/node/ledger/shims/abcipp_shim_types.rs

```diff
@@ -273,6 +273,24 @@ pub mod shim {
             types::ConsensusParams,
         };
 
+        #[derive(Debug, Default)]
+        pub struct PrepareProposal {
+            pub txs: Vec<super::TxBytes>,
+        }
+
+        #[cfg(feature = "abcipp")]
+        impl From<PrepareProposal> for ResponsePrepareProposal {
+            fn from(_: PrepareProposal) -> Self {
+                Self::default()
```

Comment:
should there be a `TODO`/`unimplemented!` here? We have https://github.com/anoma/namada/issues/198 as a catch all issue we can link to